### DISCUSSION TITLE
Expected Singleton Error

### DISCUSSION THREAD

**Discussion by it12uw** [OCA/odoo-community.org#97] [Q&A] [answered]
hello, i have a problem when i try to download multiple selected file i always got error message like this  ValueError("Expected singleton: %s" % self)
ValueError: <class 'ValueError'>: "Expected singleton: account.invoice(35590, 35589, 35588, 35587)" while evaluating
'action = records.download_efaktur()'

do you guys have any solution for me to fix this error?

thankyou

**Comment by ap-wtioit** [comment, marked as answer]
Hi @it12uw,

it looks like you are trying to report a bug. Normally it helps giving the version of Odoo you are using and at least the button you clicked.
This one unfortunately seems pretty obvious and also was reported on [stackoverflow](https://stackoverflow.com/q/76503112/925125) it seems.

It affects https://github.com/odoo/odoo/blob/16.0/addons/l10n_id_efaktur/views/account_move_views.xml#L44 at least up to 16.0. So technically you should report it in the @odoo issues https://github.com/odoo/odoo/issues and maybe ping the authors there. If you have an enterprise subscription you might wanna use the support flow there.

The workarround for this is to download the E-Faktur one by one. (Not a good workarround, i know)

Having said all this, this doesn't really seam to belong to the @OCA discussions.

regards
Andreas
